aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -0,0 +1,146 @@
+- WebSockets (Windows 8)
+
+Supported Windows versions:
+
+- Windows 7 and Windows Server 2008 R2 and later
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/servers/httpsys/sample)
+
+## When to use HttpSys
+
+HttpSys is useful for deployments where you need to expose the server directly to the Internet without using IIS.
+
+![HttpSys to Internet](httpsys/_static/httpsys-to-internet.png)
+
+Because it's built on Http.Sys, HttpSys doesn't require a reverse proxy server for protection against attacks. Http.Sys is mature technology that protects against many kinds of attacks and provides the robustness, security, and scalability of a full-featured web server. IIS itself runs as an HTTP listener on top of Http.Sys. 
+
+HttpSys is also a good choice for internal deployments when you need one of the features it offers that you can't get by using Kestrel.
+
+![HttpSys to Internet](httpsys/_static/httpsys-to-internal.png)
+
+## How to use HttpSys
+
+Here's an overview of setup tasks for the host OS and your ASP.NET Core application.
+
+### Configure Windows Server
+
+* Install the version of .NET that your application requires, such as [.NET Core](https://go.microsoft.com/fwlink/?LinkID=827524) or .NET Framework.
+
+* Preregister URL prefixes to bind to HttpSys, and set up SSL certificates
+
+   If you don't preregister URL prefixes in Windows, you have to run your application with administrator privileges. The only exception is if you bind to localhost using HTTP (not HTTPS) with a port number greater than 1024; in that case administrator privileges aren't required.
+
+   For details, see [How to preregister prefixes and configure SSL](#preregister-url-prefixes-and-configure-ssl) later in this article.
+
+* Open firewall ports to allow traffic to reach HttpSys.
+
+   You can use netsh.exe or [PowerShell cmdlets](https://technet.microsoft.com/library/jj554906).
+
+There are also [Http.Sys registry settings](https://support.microsoft.com/kb/820129).
+
+### Configure your ASP.NET Core application
+
+* No package install is needed if you use the [Microsoft.AspNetCore.All](xref:fundamentals/metapackage) metapackage. The [Microsoft.AspNetCore.Server.HttpSys](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.HttpSys/) package is included in the metapackage.
+
+* Call the [`UseHttpSys`](http://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilderKestrelExtensions/index.html#Microsoft.AspNetCore.Hosting.WebHostBuilderHttpSysExtensions.UseHttpSys.md) extension method on [WebHostBuilder](http://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html#Microsoft.AspNetCore.Hosting.WebHostBuilder.md) in your `Main` method, specifying any HttpSys [options](https://github.com/aspnet/HttpSysServer/blob/rel/2.0.0/src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs) that you need, as shown in the following example:
```

Comment:
http://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilderKestrelExtensions/index.html#Microsoft.AspNetCore.Hosting.WebHostBuilderHttpSysExtensions.UseHttpSys.md is not working for me. 

check your links that end in **.md**, they don't look right.